plugins/main/sriov/README.md

```diff
@@ -0,0 +1,69 @@
+# SR-IOV CNI plugin
+
+NIC with [SR-IOV](http://blog.scottlowe.org/2009/12/02/what-is-sr-iov/) capabilities works by introducing the idea of physical functions (PFs) and virtual functions (VFs). 
+
+PF is used by host.Each VFs can be treated as a separate physical NIC and assigned to one container, and configured with separate MAC, VLAN and IP, etc.
+
+## Enable SR-IOV
+
+Given Intel ixgbe NIC on CentOS, Fedora or RHEL:
+
+```
+# vi /etc/modprobe.conf
+options ixgbe max_vfs=8,8
```

Comment:
It's requirement. SR-IOV plugin need NIC enable SR-IOV capability, which not enabled by default.